maintenance/scripts/refresh_api.sh

```diff
@@ -1,9 +1,9 @@
 #! /bin/bash
 
-parent_path=$( cd "$(dirname "${BASH_SOURCE[0]}")" ; pwd -P )
+parent_path=~/gc-digital-talent/maintenance/scripts
 source ${parent_path}/lib/common.sh
 
-cd /var/www/html/api
+cd ~/gc-digital-talent/api
```

Comment:
Was this change on purpose or just testing? It breaks the script for me locally and seems out of scope for this issue.